### PR TITLE
bug ipinfo - not showing all info 

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46061

## Description
adding all of the info of the output for the ip command for ipinfo

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by jochman — CHANGES_REQUESTED** [review]
Please see what is the results from the original ipinfo and see if you can match it. P
When you're done attach screenshots of the changes.

Anyway, great work 🦎

File: Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py [hunk 1]
```diff
@@ -35,22 +35,22 @@ def test_ipinfo_ip_command(requests_mock):
...
+    command_results = mocker.patch('ipinfo_v2.CommandResults')
+    ipinfo_ip_command(client, ip)
 
     expected_parsed_context = util_load_json('test_data/ip_1.1.1.1_command_results.json')
-    assert parsed_context == expected_parsed_context
+    assert command_results.call_args[1].get("readable_output") == expected_parsed_context[1].get("HumanReadable")
 
 
-def test_ip_command(requests_mock):
+def test_ip_command(mocker):
```
Comment (jochman) [inline Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py:53]: Change to a meaningful name -> `test_what_case_i_am_testing`
Change the Given-When-Where to be meaningful and describe what you're testing.

File: Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py [hunk 2]
```diff
@@ -65,15 +65,14 @@ def test_ip_command(requests_mock):
...
                     reliability=DBotScoreReliability.C)
 
-    command_results = ipinfo_ip_command(client, ip)
-    parsed_context = [command_result.to_context() for command_result in command_results]
+    mocker.patch.object(client, 'http_request', return_value=mock_response)
+    command_results = mocker.patch('ipinfo_v2.CommandResults')
+    ipinfo_ip_command(client, ip)
 
-    assert "lat|lng" in parsed_context[1].get("HumanReadable")
+    assert "lat|lng" in command_results.call_args[1].get("readable_output")
```
Comment (jochman) [inline Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py:78]: That's what you're looking for in the test?

**Review by jochman — CHANGES_REQUESTED** [review]
Overall nice, I've added a comment for missing command name/docstring in the test. Please fix and let someone review the changelog.

File: Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py (hunk 2 above)
Comment (MLainer1) [reply Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py:78]: yes, my change includes those two that were missing from the table, and appeared alone in the info part
Screen Shot 2022-01-26 at 16 37 29

File: Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py (hunk 1 above)
Comment (MLainer1) [reply Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py:53]: how about test_check_columns_exists

File: Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py (hunk 2 above)
Comment (jochman) [reply Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py:78]: But this happened before. Don't you want to check that the new output is the right one?

Comment (MLainer1) [reply Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py:78]: Screen Shot 2022-01-26 at 17 06 23

this is the new one

Comment (jochman) [reply Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py:78]: cool. Is that all the difference? if not, might want to check if the ip/company in it as example.

File: Packs/ipinfo/ReleaseNotes/2_1_2.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### IPinfo v2
+- Fixed an issue where the ***ip*** command would not present all the information from the api.
```
Comment (avidan-H) [inline Packs/ipinfo/ReleaseNotes/2_1_2.md:4]:
```suggestion
- Fixed an issue where the ***ip*** command would not present all of the information returned from the API.
```

**Review by jochman — APPROVED** [review]
🚀

**Pull request merged**